packages/core/src/document/editor.tsx

```diff
@@ -104,7 +104,8 @@ export const DocumentEditor: React.FunctionComponent<
         payload: {
           id,
           state: stateHandler
-        }
+        },
+        debounce: true
```

Comment:
I'd invert the naming, i.e. `commit: true` to force committing. Debounce should be the default behavior IMHO. What's your opinion about that?

Reply:
Also: this naming "forces" us to use a debounce semantics. But IMHO, the combine semantics could be anything, really (& customizable)

Reply:
Maybe even more options in the future? E.g. `commit: true` to force comitting and `undo: false` to prevent committing the action itself (e.g. for meaningless actions).